ontologies/vocabs.ttl

```diff
@@ -1088,8 +1067,10 @@ Tag for the Education industry sector"""^^xsd:string ;
 
 gist:_IndustrySector_emergency-services
 	a gist:IndustrySector ;
-	rdfs:label "Emergency-Services"^^xsd:string ;
-	skos:altLabel "Emergency-Services"^^xsd:string ;
+	skos:altLabel
+		"Emergency Services"^^xsd:string ,
+		"Emergency-Services"^^xsd:string
```

Comment:
Is this helpful?
Also, should there not be a stixTerm? I guess that is a different git issue?